solr/solrj-streaming/src/java/org/apache/solr/client/solrj/io/eval/OutliersEvaluator.java

```diff
@@ -97,13 +97,13 @@ public Object doWork(Object... values) throws IOException {
         double cumProb = d.cumulativeProbability(n.intValue());
         if (low >= 0 && cumProb <= low) {
           t.put("lowOutlierValue_d", n);
-          t.put("cumulativeProbablity_d", cumProb);
+          t.put("cumulativeProbability_d", cumProb);
```

Comment:
I was kind of hoping that since I fixed it in hte unit test, that it would be okay.   I will says, this code, i don't know how to deprecate it and fix it in a "smooth" manner...

Reply:
Okay, going back and forth on this...   I am going to make a JIRA for this and track it as a breaking change, though I dont' think anyone uses it because it's not in the ref guide or the visual guide...

Reply:
https://issues.apache.org/jira/browse/SOLR-17425